src/app/modules/shared_models/token_model.nim

```diff
@@ -153,8 +153,8 @@ QtObject:
     case column:
       of "name": result = $item.getName()
       of "symbol": result = $item.getSymbol()
-      of "totalBalance": result = $item.getTotalBalance()
-      of "totalCurrencyBalance": result = $item.getTotalCurrencyBalance()
+      of "totalBalance": result = $item.getTotalBalance().toJsonNode()
```

Comment:
If we don't return it as JSON, it will be returned as some-formatting string, which is defined here: 
https://github.com/status-im/status-desktop/blob/8ee81f8ac70a80b1732336f131120327882a6af6/src/app/modules/shared_models/token_item.nim#L83-L107

We convert it to JSON to make it parsable in QML 🙂